includes/Admin/Orders.php

```diff
@@ -103,7 +103,45 @@ public function enqueue_assets() {
 	 * @since 2.1.0-dev.1
 	 */
 	public function add_notices() {
+		global $post;
+
+		if ( ! $this->is_edit_order_screen() ) {
+			return;
+		}
+
+		$order  = wc_get_order( $post );
+		$plugin = facebook_for_woocommerce();
+
+		if ( Commerce\Orders::is_order_pending( $order ) ) {
 
+			/* translators: Placeholders: %1$s - <strong> tag, %2$s - </strong>, %3$s - <strong> tag, %4$s - </strong> tag */
+			$message = sprintf(
+				__( 'This order is currently being held by Instagram and cannot be edited. Once released by Instagram, it will move to %1$sProcessing%2$s or %3$sCancelled%4$s status.', 'facebook-for-woocommerce' ),
+				'<strong>', '</strong>',
```

Comment:
No need to have this twice, you can use %1$s and %2$s multiple times on the text.

Reply:
Done in https://github.com/facebookincubator/facebook-for-woocommerce/pull/1576/commits/8a78e4d5de7f094320bd47353a7d82e4699d8d70.